modules/cnf-cpu-infra-container.adoc

```diff
@@ -5,17 +5,62 @@
+
+|5
+|Latency sensitive workload pods
+|Pinned to a specific set of exclusive CPUs from the isolated pool
+|===
+
+The exact partitioning pattern to use depends on many factors like hardware, workload characteristics and the expected system load. Some sample use cases are as follows:
+
+* If the latency sensitive workload uses specific hardware (like network interface cards (NICs)), ensure that the CPUs in the isolated pool are as "close" to this hardware as possible. Place the workload in at least the same Non-Uniform Memory Access (NUMA) node.
+
+* The reserved pool is used for handling all interrupts. When depending on system networking, allocate a sufficiently sized reserve pool to handle all the incoming packet interrupts. In version 4.7+, workloads can optionally be labeled as sensitive. More information regarding the management of device interrupt processing can be found in link:https://docs.openshift.com/container-platform/4.8/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus_cnf-master[Managing device interrupt processing for guaranteed pod isolated cpus].
+
+The above examples require detailed analysis and measurements to be combined into a pattern where all housekeeping CPUs are allocated from the NUMA node(s). The reserved CPUs can also be distributed among all NUMA nodes. The selection depends on the workload architecture and the specific use case.
+
+[IMPORTANT]
+====
+The reserved and isolated CPU pools must not overlap and together must span all available cores in the worker node.
+====
+
+To ensure that housekeeping tasks and workloads do not interfere with each other, specify two groups of CPUs in the `spec` section of the performance profile. 
 
 * `isolated` - The CPUs for the application container workloads. These CPUs have the lowest latency. Processes in this group have no interruptions and can, for example, reach much higher DPDK zero packet loss bandwidth.
 
-* `reserved` - The CPUs for the cluster and operating system housekeeping duties, including pod infra containers. Threads in the `reserved` group tend to be very busy, so latency-sensitive applications should be run in the `isolated` group. See link:https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed[Create a pod that gets assigned a QoS class of `Guaranteed`].
+* `reserved` - The CPUs for the cluster and operating system housekeeping duties, including pod infra containers. Threads in the `reserved` group tend to run multiple processes at random, so latency-sensitive applications should be run in the `isolated` group. See link:https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed[Create a pod that gets assigned a QoS class of `Guaranteed`].
```

Comment:
I still like the old "busy" work better :)

Reply:
Updated as suggested.